src/lib/sql/sql_translator.hpp

```diff
@@ -149,6 +159,9 @@ class SQLTranslator final {
 
   std::shared_ptr<AbstractExpression> _inverse_predicate(const AbstractExpression& expression) const;
 
+  std::vector<std::shared_ptr<AbstractExpression>> _unwrap_elements(
+      const std::vector<SelectListElement>& select_list_elements) const;
```

Comment:
I would add a comment here answering the following two questions:
1.  What does this function return? 
2. What is this function used for?

Reply:
Good idea, I also do not like the function name very much. ;)
My suggestion is:
```
// Convenience function that extracts and returns the expressions in select_list_elements
```
Now that you point on it, it does not even have to be a method of the `SQLTranslator` but can be a helper function outside of the class. If you want, you can refactor this.